src/common/search.rs

```diff
@@ -110,7 +110,7 @@ pub fn GlobalSearchBar() -> impl IntoView {
 
     view! {
         <PreSectionContainer>
-            <div class="mx-2 my-2 md:mx-0 md:w-full -mt-2 relative align-stretch flex items-center">
+            <div class="mx-2 my-2 md:mx-0 md:w-full mt-5 relative align-stretch flex items-center">
```

Comment:
This change doesn't seem related

Reply:
The search bar alignment needed to be adjusted to be inside the gray background (since it was no longer against the top.

Reply:
got it